nethvoice.rst

```diff
@@ -1330,3 +1333,186 @@ All users who have that profile will be enabled to use the newly created URL.
+
+* Service extensions, such as the reception, should not be added to the hotel profile and must be configured as standard extensions, following the hotel's usual numbering policy.
+For example, if room extensions range from 201 to 299, the reception extension should always be a three-digit number outside this range, such as 200 or 300.
+To allow rooms to call reception, a speed dial number should be configured. Service extensions, however, can call each other directly.
+
+* It is advisable to use a separate Outbound Route without a prefix for service extensions, different from the one used for room extensions.
+
+Phone Feature Codes
+-------------------
+
+In the NethVoice PBX management interface, under Service Codes, you can find the codes to use NethHotel features directly from the phones.
+For example, to add an extra charge to a room, you would dial:
+
+   ``*33 + Room Extension + # + Extra ID + # + Quantity``
+
+Room Management
+---------------
+
+On the main page, all configured extensions are listed. Rooms are divided into tabs based on the numeric value of the callgroup field of the telephone extension, which can be configured from the NethVoice interface.
+Available rooms are marked in green, the rooms that have already checked in are marked in red while te rooms that need cleaning are marked in yellow.
+
+All available functions are presented directly within the room’s panel. It is also possible to use the contextual menu by right-clicking.
+
+Wake-up Call
+^^^^^^^^^^^^
+
+The wake-up call can be scheduled either as a one-time event or repeated over multiple days. Each guest can set the wake-up call for their room by dialing the number
+
+``977``
+
+Groups
+------
+
+It is possible to group multiple rooms into a single group. This allows performing actions (check-in, check-out, wake-up call) on all rooms in the group simultaneously, and to define call policies for group members in the settings (e.g., enabling calls between rooms, between all rooms, or allowing external calls).
+
+Add an Extra
+------------
+
+To add an extra to a room, simply click the corresponding icon.
+
+Report
+------
+
+To generate a bill report for currently occupied rooms, click the related icon.
+The report includes a detailed list of all calls and extras, with a final summary showing the total amount in real time.
+
+Rates
+-----
+
+NethHotel includes a default set of call rates based on the type of call (e.g., mobile, local, etc.).
+You can modify existing rates or create new ones. It is also possible to enable or disable calls to specific number types.
+
+Extras
+------
+
+Extras can be configured within the system and later assigned to rooms either via the web interface or directly from the phone.
+
+For example, to charge three units of the extra with code 99 to room 201, simply dial:
+
+``*33201#99#3 will charge three units of the extra with code 99 to room 201``
+
+Options
+-------
+
+* The general options include:
```

Comment:
This should be a point of the list.
Please fix the formatting. See the preview to check how it looks now.